pkg/cmd/prebuild/add.go

```diff
@@ -134,8 +181,10 @@ var prebuildAddCmd = &cobra.Command{
 	},
 }
 
-var runOnAddFlag bool
-
 func init() {
 	prebuildAddCmd.Flags().BoolVar(&runOnAddFlag, "run", false, "Run the prebuild once after adding it")
+	prebuildAddCmd.Flags().StringVarP(&branchFlag, "branch", "b", "", "Git branch for the prebuild")
+	prebuildAddCmd.Flags().IntVarP(&retentionFlag, "retention", "r", 0, "Retention period for the prebuild (in days)")
```

Comment:
The description for retention should be: "Maximum number of resulting builds stored at a time"

Reply:
updated